src/main/java/lotto/LottoNumberGenerator.java

```diff
@@ -20,22 +18,16 @@ public class LottoNumberGenerator {
         }
     }
 
-    public List<Integer> autoGenerateNumbers() {
+    InputView inputView = new InputView();
+
+    public Lotto autoGenerateNumbers() {
         Collections.shuffle(lottoNumberKeys);
-        return lottoNumberKeys.stream()
+        return Lotto.from(lottoNumberKeys.stream()
                 .limit(LOTTO_NUMBER_SIZE)
-                .collect(Collectors.toList());
+                .collect(Collectors.toList()));
     }
 
-    public List<Integer> manualGenerateNumbers() {
-        final Scanner scanner = new Scanner(System.in);
-        final Spliter spliter = new Spliter();
-        Optional<String> inputOptional = Optional.ofNullable(scanner.nextLine());
-        String input = inputOptional.orElse("");
-        Validate.validateLottoNumber(input);
-        Validate.validateLottoNumberCount(input);
-        Validate.validateLottoNumberRange(input);
-        Validate.validateLottoNumberDuplicate(input);
-        return spliter.splitToList(input);
+    public Lotto manualGenerateNumbers() {
+        return inputView.inputManualLottoNumber();
```

Comment:
view에 대한 의존성을 controller가 아닌 도메인 객체가 가지고있네요..😭
저는 이런 케이스에서는 Lotto를 리턴하는 함수를 파라미터로 넘겨받아서 처리했었어요! 😀
```suggestion
    public Lotto manualGenerateNumbers(Supplier<Lotto> supplier) {
        return supplier.get();
```
이런식이 되겠네요.

혹은 LottoNumberGenerator의 상위 인터페이스를 두고 ([함수형 인터페이스](https://bcp0109.tistory.com/313))

> 1. `자동으로 Lotto를 만드는 구현클래스`를 주입
> 2. controller에서 `view를 통해 수동으로 번호를 입력받아 Lotto를 리턴하는 함수`를 주입

위 방식으로 사용할수도 있겠네요!

혹은 controller에서 수동 로또 `수량`과 `번호`를 입력받은 후 파라미터로 넘겨주는 방법도 있겠네요!

방법은 여러가지가 있을수있고 정답은 없으니 참고정도만 부탁드려요. 😀